src/ast-utils.ts

```diff
@@ -17,80 +17,106 @@ import {ASTNode, treeAdapters} from 'parse5';
 
 import * as matchers from './matchers';
 
-export function prepend(parent: ASTNode, node: ASTNode) {
-  if (parent.childNodes && parent.childNodes.length) {
-    dom5.insertBefore(parent, parent.childNodes[0], node);
-  } else {
-    dom5.append(parent, node);
-  }
-}
-
-export function prependAll(parent: ASTNode, nodes: ASTNode[]) {
-  let moveIndex = nodes.length - 1;
-  while (moveIndex >= 0) {
-    const nodeToMove = nodes[moveIndex];
-    dom5.remove(nodeToMove);
-    prepend(parent, nodeToMove);
-    moveIndex--;
-  }
+/**
+ * Move the `node` to be the immediate sibling after the `target` node.
+ * TODO(usergenic): Migrate this code to polymer/dom5
+ */
+export function insertAfter(target: ASTNode, node: ASTNode) {
+  dom5.remove(node);
+  const index = target.parentNode!.childNodes!.indexOf(target);
+  target.parentNode!.childNodes!.splice(index + 1, 0, node);
```

Comment:
Meh dom5 doesn't export insertNode today.  I updated the TODO comment on this function to make that change when porting it to dom5.